src/server.ml

```diff
@@ -80,6 +80,7 @@ module Make (IO : IO) = struct
     object
       val mutable uri = uri
       method set_uri u = uri <- Some u
```

Comment:
I'm not sure I follow :)

your change means you can _remove_ the URI from this object. I guess it can make sense(?) but I don't see why would you want to "set uri back"?